src/web-auth/captcha.js

```diff
@@ -189,6 +189,23 @@ function injectCaptchaScript(element, opts, callback, setValue) {
       window.arkose = arkose;
       callback(arkose);
     };
+  } else if (opts.provider === AUTH0_V2_CAPTCHA_PROVIDER) {
+    var a0RetryCount = 0;
```

Comment:
We shouldn't have a separate variable for Auth0 V2, we can use the retryCount variable since when this script runs there will always be one provider so the counter will only represent the retries for that provider.